### PR TITLE
refactor: Remove all usages of Bleve

### PR DESCRIPTION
**Pull request by connorgorman**
## Description

Removes all dependencies on Bleve

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

CI

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -15,7 +15,6 @@ require (
 	github.com/VividCortex/ewma v1.2.0
 	github.com/andygrunwald/go-jira v1.16.0
 	github.com/aws/aws-sdk-go v1.44.257
-	github.com/blevesearch/bleve v1.0.14
```
Comment (janisz) [inline go.mod:18]: ❤️

File: central/graphql/resolvers/vulnerability_requests_test.go [hunk 2]
```diff
@@ -228,9 +226,9 @@ func (s *VulnRequestResolverTestSuite) createComponentCVEDataStore(_ *dackbox.Da
 }
 
 func (s *VulnRequestResolverTestSuite) createVulnRequestDataStore(pendingReqCache, activeReqCache vulnReqCache.VulnReqCache) {
-	ds, err := vulnReqDS.NewForTestOnly(s.T(), s.db, s.bleveIndex, pendingReqCache, activeReqCache)
-	s.NoError(err)
-	s.vulnReqDataStore = ds
+	//ds, err := vulnReqDS.NewForTestOnly(s.T(), s.db, nil, pendingReqCache, activeReqCache)
+	//s.NoError(err)
+	//s.vulnReqDataStore = ds
```
Comment (janisz) [inline central/graphql/resolvers/vulnerability_requests_test.go:231]: why commenting and not removing?

**Review by rhybrillou — COMMENTED** [review]
Same question regarding commenting out code vs removing it. I suppose it is due to parallel work in the vulnerabilityrequest area.

**Review by md2119 — APPROVED** [review]

File: central/graphql/resolvers/vulnerability_requests_test.go (hunk 2 above)
Comment (connorgorman) [reply central/graphql/resolvers/vulnerability_requests_test.go:231]: I just commented to keep the test looking the same and not deleting. There are tickets for porting these tests to Postgres, but didn't want to just delete the code and then the test may be more difficult to port

**Pull request merged**